umbraco/Umbraco.CMS.Backoffice
# Multi Url Picker package - including markdown editor action extension point

This PR collects all functionality related to the "Mult URL Picker" and adds it as a new "package".
* Property Editors
* Tiny MCE Plugin
* Markdown Editor plugin

To make it possible to do this I had to introduce an extension point to the markdown editor. It is currently only the multi url picker that registers an extension. We will have to change the other actions later. 

How to test:
* Make sure you can load the multi url picker property editor
* Make sure you can use the link picker in Tiny MCE
* Make sure you can use the link picker in the Markdown editor.
